src/sparql/mondo_stats/COUNT-xrefs.sparql

```diff
@@ -0,0 +1,20 @@
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+# Count the total number of xrefs on unique non-obsolete Mondo classes 
+
+SELECT (COUNT(DISTINCT ?xref) AS ?xrefCount)
+WHERE {
+  ?class rdfs:subClassOf* obo:MONDO_0000001 .
+  
+  FILTER NOT EXISTS {
+    ?class owl:deprecated "true"^^xsd:boolean
+  }
+  
+  ?class oboInOwl:hasDbXref ?xref .
```

Comment:
Stylistically, as with the other SPARQL files here too, should this instead be in a single clause?

```sparql
  ?class rdfs:subClassOf* obo:MONDO_0000001 ;
      oboInOwl:hasDbXref ?xref .
```

Reply:
updated